bot/messages.js

```diff
@@ -53,6 +57,7 @@ const invoicePaymentRequestMessage = async (
 ) => {
   try {
     let currency = getCurrency(order.fiat_code);
+    const buyerResponse = await calculateBuyerResponse(order.buyer_id);
```

Comment:
We shouldn't make calculations on every new publish order, this will make this process slow when users have more responses to calculate